sw/device/tock/dif/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Bindings for OpenTitan DIF libaries
+
+#![no_std]
+#![crate_name = "dif"]
+#![crate_type = "rlib"]
+
+#[allow(nonstandard_style)]
+pub mod dif_bind;
+#[allow(nonstandard_style)]
```

Comment:
I would move this into the module file, and then mark each type explicitly with it.

Reply:
What is your reasoning here? In case the module file contains some standard types?

Reply:
Done.